themes/default/table.css

```diff
@@ -0,0 +1,7 @@
+.ct-table-container {
+    margin: 20px 0 5px;
+}
+
+.ct-table {
+  width: 100%;
+}
```

Comment:
If I'm not mistaking, the build will actually fail from this, right? A new line at the end of a file is MANDATORY. :)